src/components/FormPost.js

```diff
@@ -0,0 +1,117 @@
+/* eslint-disable no-unused-vars */
+import React, { useState } from 'react';
+import post from '../utils/post';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import FormButtons from './FormButtons';
+import Tags from '../utils/Tags';
+import Typography from '@material-ui/core/Typography';
+import getPosses from '../utils/getPosses';
+
+function FormPost({dialogOpen}) {
+  const [tags, setTags] = useState(Tags);
+  const [posses, setPosses] = useState([]);
+  const [postTxt, setPostTxt] = useState('');
+  let globalPosse = {};
+
+  useState(() => {
+    getPosses()
+      .then(res => {
+        if (res.data) {
+          const posses = res.data.results;

```

Comment:
true that a good point but we only using that const posses to get the new and since the scoping of that const will only live inside of that if(res.data) statement it shouldn't be a problem but you are right it would be better to change it.